pkg/spec/spec.go

```diff
@@ -126,7 +152,7 @@ func CreateConfigToOCISpec(config *CreateConfig) (*spec.Spec, error) { //nolint
 		g.RemoveMount("/dev/mqueue")
 		devMqueue := spec.Mount{
 			Destination: "/dev/mqueue",
-			Type:        "bind",
+			Type:        bindMount,
```

Comment:
we could probably just drop the `Type` altogether.  The OCI runtime looks at the options to see if it is a bind mount (either contains `bind` or `rbind`)

Reply:
I'll leave it for now, it's in the official OCI spec.